.github/workflows/nodejs.yml

```diff
@@ -100,15 +100,13 @@ jobs:
       matrix:
         node-version: [14.x, 16.x, 18.x]
 
-    services:
-      rippled:
-        image: natenichols/rippled-standalone:latest
-        ports:
-          - 6006:6006
-        options: --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s
-
     steps:
       - uses: actions/checkout@v3
+
+      - name: Run docker in background
+        run: |
+          docker run -d --rm --name rippled-service -p 6006:6006 -v "${{ github.workspace }}/ci-config/":"/config/" --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s -e "ENV_ARGS=-a --start" -e GITHUB_ACTIONS=true -e CI=true xrpllabsofficial/xrpld:latest
```

Comment:
For the purpose of readability, can you use the long version of the flags? e.g. `--detach` instead of `-d`

Reply:
Yep, except for `-p` because it's long name is `--expose` which seems more confusing than `-p` to me :)